src/theory/inference_id.h

```diff
@@ -462,8 +462,6 @@ enum class InferenceId
   SEP_LABEL_DEF,
   // lemma for sep.emp
   SEP_EMP,
-  // lemma for positive labelled PTO
-  SEP_POS_PTO_SINGLETON,
```

Comment:
Indeed the code in preNotifyFact is redundant. But it seems SEP_POS_REDUCTION handles a lot of inference cases. 
Wouldn't using this inference Id [here](https://github.com/cvc5/cvc5/blob/main/src/theory/sep/theory_sep.cpp#L424)  be more fine-grained?

Reply:
I would only make this a different inference id if the lemma were called at a different place in the code.